<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4 Electrical Basis Of Neuronal Function | Text For BIO400 Neuroanatomy at Salem State University</title>
  <meta name="description" content="This is a free online Text for Neuroanatomy" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="4 Electrical Basis Of Neuronal Function | Text For BIO400 Neuroanatomy at Salem State University" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="cover.jpg" />
  <meta property="og:description" content="This is a free online Text for Neuroanatomy" />
  <meta name="github-repo" content="nikolaussucher/neuroanatomy-text" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4 Electrical Basis Of Neuronal Function | Text For BIO400 Neuroanatomy at Salem State University" />
  <meta name="twitter:site" content="@NikolausSucher" />
  <meta name="twitter:description" content="This is a free online Text for Neuroanatomy" />
  <meta name="twitter:image" content="cover.jpg" />

<meta name="author" content="Assembled by Nikolaus Sucher" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="neurons-and-glial-cells.html"/>
<link rel="next" href="neurotransmission.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<link rel="stylesheet" href="css/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Neuroanatomy</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="the-nervous-system.html"><a href="the-nervous-system.html"><i class="fa fa-check"></i><b>1</b> The Nervous System</a><ul>
<li class="chapter" data-level="1.1" data-path="the-nervous-system.html"><a href="the-nervous-system.html#introduction"><i class="fa fa-check"></i><b>1.1</b> Introduction</a></li>
<li class="chapter" data-level="1.2" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-cells-of-the-nervous-system"><i class="fa fa-check"></i><b>1.2</b> The Cells Of The Nervous System</a></li>
<li class="chapter" data-level="1.3" data-path="the-nervous-system.html"><a href="the-nervous-system.html#comparative-anatomy-and-evolution-of-nervous-systems"><i class="fa fa-check"></i><b>1.3</b> Comparative Anatomy And Evolution Of Nervous Systems</a></li>
<li class="chapter" data-level="1.4" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-human-brain"><i class="fa fa-check"></i><b>1.4</b> The Human Brain</a></li>
<li class="chapter" data-level="1.5" data-path="the-nervous-system.html"><a href="the-nervous-system.html#development-of-the-nervous-system"><i class="fa fa-check"></i><b>1.5</b> Development Of The Nervous System</a></li>
<li class="chapter" data-level="1.6" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-function-of-the-nervous-system"><i class="fa fa-check"></i><b>1.6</b> The Function Of The Nervous System</a></li>
<li class="chapter" data-level="1.7" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-sensory-system"><i class="fa fa-check"></i><b>1.7</b> The Sensory System</a></li>
<li class="chapter" data-level="1.8" data-path="the-nervous-system.html"><a href="the-nervous-system.html#the-motor-system"><i class="fa fa-check"></i><b>1.8</b> The Motor System</a></li>
<li class="chapter" data-level="1.9" data-path="the-nervous-system.html"><a href="the-nervous-system.html#neuronal-signalling"><i class="fa fa-check"></i><b>1.9</b> Neuronal Signalling</a></li>
<li class="chapter" data-level="1.10" data-path="the-nervous-system.html"><a href="the-nervous-system.html#neural-circuits"><i class="fa fa-check"></i><b>1.10</b> Neural Circuits</a></li>
<li class="chapter" data-level="1.11" data-path="the-nervous-system.html"><a href="the-nervous-system.html#reflexes-and-other-stimulus-response-circuits"><i class="fa fa-check"></i><b>1.11</b> Reflexes And Other Stimulus-Response Circuits</a></li>
<li class="chapter" data-level="1.12" data-path="the-nervous-system.html"><a href="the-nervous-system.html#intrinsic-pattern-generation"><i class="fa fa-check"></i><b>1.12</b> Intrinsic Pattern Generation</a></li>
<li class="chapter" data-level="1.13" data-path="the-nervous-system.html"><a href="the-nervous-system.html#neuroimaging"><i class="fa fa-check"></i><b>1.13</b> Neuroimaging</a><ul>
<li class="chapter" data-level="1.13.1" data-path="the-nervous-system.html"><a href="the-nervous-system.html#electroencephalography"><i class="fa fa-check"></i><b>1.13.1</b> Electroencephalography</a></li>
<li class="chapter" data-level="1.13.2" data-path="the-nervous-system.html"><a href="the-nervous-system.html#magnetoencephalography"><i class="fa fa-check"></i><b>1.13.2</b> Magnetoencephalography</a></li>
<li class="chapter" data-level="1.13.3" data-path="the-nervous-system.html"><a href="the-nervous-system.html#computed-axial-tomography"><i class="fa fa-check"></i><b>1.13.3</b> Computed axial tomography</a></li>
<li class="chapter" data-level="1.13.4" data-path="the-nervous-system.html"><a href="the-nervous-system.html#magnetic-resonance-imaging"><i class="fa fa-check"></i><b>1.13.4</b> Magnetic resonance imaging</a></li>
<li class="chapter" data-level="1.13.5" data-path="the-nervous-system.html"><a href="the-nervous-system.html#positron-emission-tomography"><i class="fa fa-check"></i><b>1.13.5</b> Positron emission tomography</a></li>
<li class="chapter" data-level="1.13.6" data-path="the-nervous-system.html"><a href="the-nervous-system.html#single-photon-emission-computed-tomography"><i class="fa fa-check"></i><b>1.13.6</b> Single-photon emission computed tomography</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html"><i class="fa fa-check"></i><b>2</b> Development Of The Nervous System</a><ul>
<li class="chapter" data-level="2.1" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#neural-induction"><i class="fa fa-check"></i><b>2.1</b> Neural Induction</a></li>
<li class="chapter" data-level="2.2" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#the-early-brain"><i class="fa fa-check"></i><b>2.2</b> The Early Brain</a></li>
<li class="chapter" data-level="2.3" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#patterning-of-the-nervous-system"><i class="fa fa-check"></i><b>2.3</b> Patterning Of The Nervous System</a></li>
<li class="chapter" data-level="2.4" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#neurogenesis"><i class="fa fa-check"></i><b>2.4</b> Neurogenesis</a></li>
<li class="chapter" data-level="2.5" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#neuronal-migration"><i class="fa fa-check"></i><b>2.5</b> Neuronal Migration</a></li>
<li class="chapter" data-level="2.6" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#neurotrophic-factors"><i class="fa fa-check"></i><b>2.6</b> Neurotrophic Factors</a></li>
<li class="chapter" data-level="2.7" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#activity-dependent-mechanisms-in-the-assembly-of-neural-circuits"><i class="fa fa-check"></i><b>2.7</b> Activity Dependent Mechanisms In The Assembly Of Neural Circuits</a></li>
<li class="chapter" data-level="2.8" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#critical-periods"><i class="fa fa-check"></i><b>2.8</b> Critical Periods</a><ul>
<li class="chapter" data-level="2.8.1" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#activity-dependent-competition"><i class="fa fa-check"></i><b>2.8.1</b> Activity-dependent competition</a></li>
</ul></li>
<li class="chapter" data-level="2.9" data-path="development-of-the-nervous-system-1.html"><a href="development-of-the-nervous-system-1.html#brain-plasticity"><i class="fa fa-check"></i><b>2.9</b> Brain plasticity</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="neurons-and-glial-cells.html"><a href="neurons-and-glial-cells.html"><i class="fa fa-check"></i><b>3</b> Neurons And Glial Cells</a><ul>
<li class="chapter" data-level="3.1" data-path="neurons-and-glial-cells.html"><a href="neurons-and-glial-cells.html#neurons"><i class="fa fa-check"></i><b>3.1</b> Neurons</a></li>
<li class="chapter" data-level="3.2" data-path="neurons-and-glial-cells.html"><a href="neurons-and-glial-cells.html#glia"><i class="fa fa-check"></i><b>3.2</b> Glia</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html"><i class="fa fa-check"></i><b>4</b> Electrical Basis Of Neuronal Function</a><ul>
<li class="chapter" data-level="4.1" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#the-membrane-potential"><i class="fa fa-check"></i><b>4.1</b> The Membrane Potential</a></li>
<li class="chapter" data-level="4.2" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#ions-and-the-forces-driving-their-motion"><i class="fa fa-check"></i><b>4.2</b> Ions And The Forces Driving Their Motion</a></li>
<li class="chapter" data-level="4.3" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#the-plasma-membrane"><i class="fa fa-check"></i><b>4.3</b> The Plasma Membrane</a></li>
<li class="chapter" data-level="4.4" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#ion-pumps"><i class="fa fa-check"></i><b>4.4</b> Ion Pumps</a></li>
<li class="chapter" data-level="4.5" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#ion-channels"><i class="fa fa-check"></i><b>4.5</b> Ion Channels</a></li>
<li class="chapter" data-level="4.6" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#the-reversal-potential"><i class="fa fa-check"></i><b>4.6</b> The Reversal Potential</a></li>
<li class="chapter" data-level="4.7" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#the-resting-potential"><i class="fa fa-check"></i><b>4.7</b> The Resting Potential</a></li>
<li class="chapter" data-level="4.8" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#the-action-potential"><i class="fa fa-check"></i><b>4.8</b> The Action Potential</a></li>
<li class="chapter" data-level="4.9" data-path="electrical-basis-of-neuronal-function.html"><a href="electrical-basis-of-neuronal-function.html#graded-potentials"><i class="fa fa-check"></i><b>4.9</b> Graded Potentials</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="neurotransmission.html"><a href="neurotransmission.html"><i class="fa fa-check"></i><b>5</b> Neurotransmission</a><ul>
<li class="chapter" data-level="5.1" data-path="neurotransmission.html"><a href="neurotransmission.html#the-synapse"><i class="fa fa-check"></i><b>5.1</b> The Synapse</a></li>
<li class="chapter" data-level="5.2" data-path="neurotransmission.html"><a href="neurotransmission.html#neurotransmitters"><i class="fa fa-check"></i><b>5.2</b> Neurotransmitters</a><ul>
<li class="chapter" data-level="5.2.1" data-path="neurotransmission.html"><a href="neurotransmission.html#modulatory-neurotransmitter-systems"><i class="fa fa-check"></i><b>5.2.1</b> Modulatory Neurotransmitter Systems</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="neurotransmission.html"><a href="neurotransmission.html#neurotransmitter-receptors"><i class="fa fa-check"></i><b>5.3</b> Neurotransmitter Receptors</a><ul>
<li class="chapter" data-level="5.3.1" data-path="neurotransmission.html"><a href="neurotransmission.html#ionotropic-receptors-neurotransmitter-gated-ion-channels"><i class="fa fa-check"></i><b>5.3.1</b> Ionotropic Receptors: Neurotransmitter-Gated Ion Channels</a></li>
<li class="chapter" data-level="5.3.2" data-path="neurotransmission.html"><a href="neurotransmission.html#metabotropic-receptors-g-protein-coupled-receptors"><i class="fa fa-check"></i><b>5.3.2</b> Metabotropic Receptors: G-Protein Coupled Receptors</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html"><i class="fa fa-check"></i><b>6</b> The Central Nervous System</a><ul>
<li class="chapter" data-level="6.1" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-meninges"><i class="fa fa-check"></i><b>6.1</b> The Meninges</a><ul>
<li class="chapter" data-level="6.1.1" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#dura-mater"><i class="fa fa-check"></i><b>6.1.1</b> Dura Mater</a></li>
<li class="chapter" data-level="6.1.2" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#arachnoid-mater"><i class="fa fa-check"></i><b>6.1.2</b> Arachnoid Mater</a></li>
<li class="chapter" data-level="6.1.3" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#pia-mater"><i class="fa fa-check"></i><b>6.1.3</b> Pia Mater</a></li>
<li class="chapter" data-level="6.1.4" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#subarachnoid-spaces"><i class="fa fa-check"></i><b>6.1.4</b> Subarachnoid Spaces</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-ventricular-system"><i class="fa fa-check"></i><b>6.2</b> The Ventricular System</a></li>
<li class="chapter" data-level="6.3" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-cerebrospinal-fluid"><i class="fa fa-check"></i><b>6.3</b> The Cerebrospinal Fluid</a></li>
<li class="chapter" data-level="6.4" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-blood-brain-barrier"><i class="fa fa-check"></i><b>6.4</b> The Blood-Brain-Barrier</a><ul>
<li class="chapter" data-level="6.4.1" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#circumventricular-organs"><i class="fa fa-check"></i><b>6.4.1</b> Circumventricular Organs</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-cerebrum"><i class="fa fa-check"></i><b>6.5</b> The Cerebrum</a><ul>
<li class="chapter" data-level="6.5.1" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-cerebral-cortex"><i class="fa fa-check"></i><b>6.5.1</b> The Cerebral Cortex</a></li>
<li class="chapter" data-level="6.5.2" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-basal-ganglia"><i class="fa fa-check"></i><b>6.5.2</b> The Basal Ganglia</a></li>
<li class="chapter" data-level="6.5.3" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-limbic-lobe"><i class="fa fa-check"></i><b>6.5.3</b> The Limbic Lobe</a></li>
<li class="chapter" data-level="6.5.4" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-hippocampus"><i class="fa fa-check"></i><b>6.5.4</b> The Hippocampus</a></li>
<li class="chapter" data-level="6.5.5" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-amygdala"><i class="fa fa-check"></i><b>6.5.5</b> The Amygdala</a></li>
<li class="chapter" data-level="6.5.6" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-thalamus"><i class="fa fa-check"></i><b>6.5.6</b> The Thalamus</a></li>
<li class="chapter" data-level="6.5.7" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-hypothalamus"><i class="fa fa-check"></i><b>6.5.7</b> The Hypothalamus</a></li>
<li class="chapter" data-level="6.5.8" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-brainstem"><i class="fa fa-check"></i><b>6.5.8</b> The Brainstem</a></li>
<li class="chapter" data-level="6.5.9" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-midbrain"><i class="fa fa-check"></i><b>6.5.9</b> The Midbrain</a></li>
<li class="chapter" data-level="6.5.10" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-cerebellum"><i class="fa fa-check"></i><b>6.5.10</b> The Cerebellum</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-spinal-cord"><i class="fa fa-check"></i><b>6.6</b> The Spinal cord</a></li>
<li class="chapter" data-level="6.7" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#central-neural-pathways"><i class="fa fa-check"></i><b>6.7</b> Central Neural Pathways</a><ul>
<li class="chapter" data-level="6.7.1" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-corpus-callosum"><i class="fa fa-check"></i><b>6.7.1</b> The Corpus Callosum</a></li>
<li class="chapter" data-level="6.7.2" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-cerebral-peduncles"><i class="fa fa-check"></i><b>6.7.2</b> The Cerebral Peduncles</a></li>
<li class="chapter" data-level="6.7.3" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-pyramidal-tracts"><i class="fa fa-check"></i><b>6.7.3</b> The Pyramidal Tracts</a></li>
<li class="chapter" data-level="6.7.4" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-corticopontine-fibers"><i class="fa fa-check"></i><b>6.7.4</b> The Corticopontine Fibers</a></li>
<li class="chapter" data-level="6.7.5" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-spinal-tracts"><i class="fa fa-check"></i><b>6.7.5</b> The Spinal Tracts</a></li>
<li class="chapter" data-level="6.7.6" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-spinocerebellar-tracts"><i class="fa fa-check"></i><b>6.7.6</b> The Spinocerebellar Tracts</a></li>
<li class="chapter" data-level="6.7.7" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#default-mode-network"><i class="fa fa-check"></i><b>6.7.7</b> Default mode network</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#sensory-maps"><i class="fa fa-check"></i><b>6.8</b> Sensory Maps</a></li>
<li class="chapter" data-level="6.9" data-path="the-central-nervous-system.html"><a href="the-central-nervous-system.html#the-cortical-homunculus"><i class="fa fa-check"></i><b>6.9</b> The Cortical Homunculus</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html"><i class="fa fa-check"></i><b>7</b> The Peripheral Nervous System</a><ul>
<li class="chapter" data-level="7.1" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#the-somatic-nervous-system"><i class="fa fa-check"></i><b>7.1</b> The Somatic Nervous System</a></li>
<li class="chapter" data-level="7.2" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#the-cranial-nerves"><i class="fa fa-check"></i><b>7.2</b> The Cranial Nerves</a><ul>
<li class="chapter" data-level="7.2.1" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#smell-i"><i class="fa fa-check"></i><b>7.2.1</b> Smell (I)</a></li>
<li class="chapter" data-level="7.2.2" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#vision-ii"><i class="fa fa-check"></i><b>7.2.2</b> Vision (II)</a></li>
<li class="chapter" data-level="7.2.3" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#eye-movement-iii-iv-vi"><i class="fa fa-check"></i><b>7.2.3</b> Eye movement (III, IV, VI)</a></li>
<li class="chapter" data-level="7.2.4" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#trigeminal-nerve-v"><i class="fa fa-check"></i><b>7.2.4</b> Trigeminal nerve (V)</a></li>
<li class="chapter" data-level="7.2.5" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#facial-expression-vii"><i class="fa fa-check"></i><b>7.2.5</b> Facial expression (VII)</a></li>
<li class="chapter" data-level="7.2.6" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#hearing-and-balance-viii"><i class="fa fa-check"></i><b>7.2.6</b> Hearing and balance (VIII)</a></li>
<li class="chapter" data-level="7.2.7" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#oral-sensation-taste-and-salivation-ix"><i class="fa fa-check"></i><b>7.2.7</b> Oral Sensation, Taste, And Salivation (Ix)</a></li>
<li class="chapter" data-level="7.2.8" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#vagus-nerve-x"><i class="fa fa-check"></i><b>7.2.8</b> Vagus Nerve (X)</a></li>
<li class="chapter" data-level="7.2.9" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#shoulder-elevation-and-head-turning-xi"><i class="fa fa-check"></i><b>7.2.9</b> Shoulder Elevation And Head-Turning (XI)</a></li>
<li class="chapter" data-level="7.2.10" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#tongue-movement-xii"><i class="fa fa-check"></i><b>7.2.10</b> Tongue Movement (XII)</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#the-spinal-nerves"><i class="fa fa-check"></i><b>7.3</b> The Spinal Nerves</a><ul>
<li class="chapter" data-level="7.3.1" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#cervical-spinal-nerves-c1c4"><i class="fa fa-check"></i><b>7.3.1</b> Cervical Spinal Nerves (C1–C4)</a></li>
<li class="chapter" data-level="7.3.2" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#brachial-plexus-c5t1"><i class="fa fa-check"></i><b>7.3.2</b> Brachial Plexus (C5–T1)</a></li>
<li class="chapter" data-level="7.3.3" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#lumbosacral-plexus-l1co1"><i class="fa fa-check"></i><b>7.3.3</b> Lumbosacral Plexus (L1–Co1)</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#the-autonomic-nervous-system"><i class="fa fa-check"></i><b>7.4</b> The Autonomic Nervous System</a></li>
<li class="chapter" data-level="7.5" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#the-sympathetic-nervous-system"><i class="fa fa-check"></i><b>7.5</b> The Sympathetic Nervous System</a></li>
<li class="chapter" data-level="7.6" data-path="the-peripheral-nervous-system.html"><a href="the-peripheral-nervous-system.html#the-parasympathetic-nervous-system"><i class="fa fa-check"></i><b>7.6</b> The Parasympathetic Nervous System</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html"><i class="fa fa-check"></i><b>8</b> The Somatic Sensory System</a><ul>
<li class="chapter" data-level="8.1" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html#touch"><i class="fa fa-check"></i><b>8.1</b> Touch</a></li>
<li class="chapter" data-level="8.2" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html#cutaneous-mechanoreceptors"><i class="fa fa-check"></i><b>8.2</b> Cutaneous Mechanoreceptors</a></li>
<li class="chapter" data-level="8.3" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html#nociception"><i class="fa fa-check"></i><b>8.3</b> Nociception</a></li>
<li class="chapter" data-level="8.4" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html#proprioception"><i class="fa fa-check"></i><b>8.4</b> Proprioception</a></li>
<li class="chapter" data-level="8.5" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html#muscle-spindles"><i class="fa fa-check"></i><b>8.5</b> Muscle Spindles</a></li>
<li class="chapter" data-level="8.6" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html#golgi-tendon-organ"><i class="fa fa-check"></i><b>8.6</b> Golgi Tendon Organ</a></li>
<li class="chapter" data-level="8.7" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html#the-somatosensory-pathways"><i class="fa fa-check"></i><b>8.7</b> The Somatosensory Pathways</a></li>
<li class="chapter" data-level="8.8" data-path="the-somatic-sensory-system.html"><a href="the-somatic-sensory-system.html#the-primary-somatosensory-cortex"><i class="fa fa-check"></i><b>8.8</b> The Primary Somatosensory Cortex</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="the-visual-system.html"><a href="the-visual-system.html"><i class="fa fa-check"></i><b>9</b> The Visual System</a><ul>
<li class="chapter" data-level="9.1" data-path="the-visual-system.html"><a href="the-visual-system.html#the-eye"><i class="fa fa-check"></i><b>9.1</b> The Eye</a></li>
<li class="chapter" data-level="9.2" data-path="the-visual-system.html"><a href="the-visual-system.html#the-retina"><i class="fa fa-check"></i><b>9.2</b> The Retina</a></li>
<li class="chapter" data-level="9.3" data-path="the-visual-system.html"><a href="the-visual-system.html#the-photoreceptors"><i class="fa fa-check"></i><b>9.3</b> The Photoreceptors</a></li>
<li class="chapter" data-level="9.4" data-path="the-visual-system.html"><a href="the-visual-system.html#visual-phototransduction"><i class="fa fa-check"></i><b>9.4</b> Visual Phototransduction</a></li>
<li class="chapter" data-level="9.5" data-path="the-visual-system.html"><a href="the-visual-system.html#the-visual-pathways"><i class="fa fa-check"></i><b>9.5</b> The Visual Pathways</a><ul>
<li class="chapter" data-level="9.5.1" data-path="the-visual-system.html"><a href="the-visual-system.html#the-optic-nerve-and-optic-tract"><i class="fa fa-check"></i><b>9.5.1</b> The Optic Nerve And Optic Tract</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="the-visual-system.html"><a href="the-visual-system.html#the-superior-colliculus"><i class="fa fa-check"></i><b>9.6</b> The Superior Colliculus</a></li>
<li class="chapter" data-level="9.7" data-path="the-visual-system.html"><a href="the-visual-system.html#the-lateral-geniculate-nucleus-lgn"><i class="fa fa-check"></i><b>9.7</b> The Lateral Geniculate Nucleus (LGN)</a></li>
<li class="chapter" data-level="9.8" data-path="the-visual-system.html"><a href="the-visual-system.html#the-visual-cortex"><i class="fa fa-check"></i><b>9.8</b> The Visual Cortex</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html"><i class="fa fa-check"></i><b>10</b> The Auditory And Vestibular Systems</a><ul>
<li class="chapter" data-level="10.1" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-ear"><i class="fa fa-check"></i><b>10.1</b> The Ear</a></li>
<li class="chapter" data-level="10.2" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-auditory-system"><i class="fa fa-check"></i><b>10.2</b> The Auditory System</a><ul>
<li class="chapter" data-level="10.2.1" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#organ-of-corti"><i class="fa fa-check"></i><b>10.2.1</b> Organ Of Corti</a></li>
<li class="chapter" data-level="10.2.2" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#auditory-transduction"><i class="fa fa-check"></i><b>10.2.2</b> Auditory Transduction</a></li>
<li class="chapter" data-level="10.2.3" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#auditory-pathways"><i class="fa fa-check"></i><b>10.2.3</b> Auditory Pathways</a></li>
<li class="chapter" data-level="10.2.4" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-cochlear-nucleus"><i class="fa fa-check"></i><b>10.2.4</b> The Cochlear Nucleus</a></li>
<li class="chapter" data-level="10.2.5" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-trapezoid-body"><i class="fa fa-check"></i><b>10.2.5</b> The Trapezoid Body</a></li>
<li class="chapter" data-level="10.2.6" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-superior-olivary-complex"><i class="fa fa-check"></i><b>10.2.6</b> The superior olivary complex</a></li>
<li class="chapter" data-level="10.2.7" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-lateral-lemniscus"><i class="fa fa-check"></i><b>10.2.7</b> The Lateral Lemniscus</a></li>
<li class="chapter" data-level="10.2.8" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-inferior-colliculi"><i class="fa fa-check"></i><b>10.2.8</b> The Inferior Colliculi</a></li>
<li class="chapter" data-level="10.2.9" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-medial-geniculate-nucleus-mgn"><i class="fa fa-check"></i><b>10.2.9</b> The Medial Geniculate Nucleus (MGN)</a></li>
<li class="chapter" data-level="10.2.10" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-primary-auditory-cortex"><i class="fa fa-check"></i><b>10.2.10</b> The Primary Auditory Cortex</a></li>
<li class="chapter" data-level="10.2.11" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-auditory-ventral-and-dorsal-streams"><i class="fa fa-check"></i><b>10.2.11</b> The Auditory Ventral And Dorsal Streams</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-vestibular-system"><i class="fa fa-check"></i><b>10.3</b> The Vestibular System</a><ul>
<li class="chapter" data-level="10.3.1" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-semicircular-canals"><i class="fa fa-check"></i><b>10.3.1</b> The Semicircular Canals</a></li>
<li class="chapter" data-level="10.3.2" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#the-otolithic-organs"><i class="fa fa-check"></i><b>10.3.2</b> The Otolithic Organs</a></li>
<li class="chapter" data-level="10.3.3" data-path="the-auditory-and-vestibular-systems.html"><a href="the-auditory-and-vestibular-systems.html#vestibular-pathways"><i class="fa fa-check"></i><b>10.3.3</b> Vestibular Pathways</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="11" data-path="the-olfactory-system.html"><a href="the-olfactory-system.html"><i class="fa fa-check"></i><b>11</b> The Olfactory System</a><ul>
<li class="chapter" data-level="11.1" data-path="the-olfactory-system.html"><a href="the-olfactory-system.html#the-nose"><i class="fa fa-check"></i><b>11.1</b> The Nose</a></li>
<li class="chapter" data-level="11.2" data-path="the-olfactory-system.html"><a href="the-olfactory-system.html#olfactory-sensory-neurons"><i class="fa fa-check"></i><b>11.2</b> Olfactory Sensory Neurons</a></li>
<li class="chapter" data-level="11.3" data-path="the-olfactory-system.html"><a href="the-olfactory-system.html#the-olfactory-bulb"><i class="fa fa-check"></i><b>11.3</b> The Olfactory Bulb</a></li>
<li class="chapter" data-level="11.4" data-path="the-olfactory-system.html"><a href="the-olfactory-system.html#the-olfactory-cortex"><i class="fa fa-check"></i><b>11.4</b> The Olfactory Cortex</a></li>
<li class="chapter" data-level="11.5" data-path="the-olfactory-system.html"><a href="the-olfactory-system.html#olfactory-pathways"><i class="fa fa-check"></i><b>11.5</b> Olfactory Pathways</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html"><i class="fa fa-check"></i><b>12</b> The Gustatory System</a><ul>
<li class="chapter" data-level="12.1" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#the-tongue"><i class="fa fa-check"></i><b>12.1</b> The Tongue</a></li>
<li class="chapter" data-level="12.2" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#the-five-basic-tastes"><i class="fa fa-check"></i><b>12.2</b> The Five Basic Tastes</a><ul>
<li class="chapter" data-level="12.2.1" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#sweetness"><i class="fa fa-check"></i><b>12.2.1</b> Sweetness</a></li>
<li class="chapter" data-level="12.2.2" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#sourness"><i class="fa fa-check"></i><b>12.2.2</b> Sourness</a></li>
<li class="chapter" data-level="12.2.3" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#saltiness"><i class="fa fa-check"></i><b>12.2.3</b> Saltiness</a></li>
<li class="chapter" data-level="12.2.4" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#bitterness"><i class="fa fa-check"></i><b>12.2.4</b> Bitterness</a></li>
<li class="chapter" data-level="12.2.5" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#savoriness-umami"><i class="fa fa-check"></i><b>12.2.5</b> Savoriness (Umami)</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#the-taste-receptors"><i class="fa fa-check"></i><b>12.3</b> The Taste Receptors</a></li>
<li class="chapter" data-level="12.4" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#the-gustatory-nucleus"><i class="fa fa-check"></i><b>12.4</b> The Gustatory Nucleus</a></li>
<li class="chapter" data-level="12.5" data-path="the-gustatory-system.html"><a href="the-gustatory-system.html#the-gustatory-cortex"><i class="fa fa-check"></i><b>12.5</b> The Gustatory Cortex</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html"><i class="fa fa-check"></i><b>13</b> The Somatic Motor System</a><ul>
<li class="chapter" data-level="13.1" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#skeletal-muscle"><i class="fa fa-check"></i><b>13.1</b> Skeletal Muscle</a></li>
<li class="chapter" data-level="13.2" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-muscle-fiber"><i class="fa fa-check"></i><b>13.2</b> The Muscle Fiber</a></li>
<li class="chapter" data-level="13.3" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-motor-unit"><i class="fa fa-check"></i><b>13.3</b> The Motor Unit</a></li>
<li class="chapter" data-level="13.4" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#somatic-motor-neurons"><i class="fa fa-check"></i><b>13.4</b> Somatic Motor Neurons</a></li>
<li class="chapter" data-level="13.5" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#neuromuscular-junction"><i class="fa fa-check"></i><b>13.5</b> Neuromuscular Junction</a><ul>
<li class="chapter" data-level="13.5.1" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#muscle-contraction"><i class="fa fa-check"></i><b>13.5.1</b> Muscle Contraction</a></li>
<li class="chapter" data-level="13.5.2" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#sliding-filament-theory"><i class="fa fa-check"></i><b>13.5.2</b> Sliding filament theory</a></li>
<li class="chapter" data-level="13.5.3" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#crossbridge-cycling"><i class="fa fa-check"></i><b>13.5.3</b> Crossbridge cycling</a></li>
<li class="chapter" data-level="13.5.4" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-stretch-reflex"><i class="fa fa-check"></i><b>13.5.4</b> The Stretch Reflex</a></li>
</ul></li>
<li class="chapter" data-level="13.6" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-motor-cortex"><i class="fa fa-check"></i><b>13.6</b> The Motor Cortex</a><ul>
<li class="chapter" data-level="13.6.1" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-supplementary-motor-cortex"><i class="fa fa-check"></i><b>13.6.1</b> The Supplementary Motor Cortex</a></li>
</ul></li>
<li class="chapter" data-level="13.7" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-basal-ganglia-1"><i class="fa fa-check"></i><b>13.7</b> The Basal Ganglia</a></li>
<li class="chapter" data-level="13.8" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-cortico-basal-ganglia-thalamo-cortical-loop"><i class="fa fa-check"></i><b>13.8</b> The Cortico-Basal Ganglia-Thalamo-Cortical Loop</a></li>
<li class="chapter" data-level="13.9" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-pyramidal-motor-system"><i class="fa fa-check"></i><b>13.9</b> The Pyramidal Motor System</a><ul>
<li class="chapter" data-level="13.9.1" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-corticospinal-tract-1"><i class="fa fa-check"></i><b>13.9.1</b> The Corticospinal Tract</a></li>
<li class="chapter" data-level="13.9.2" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-corticobulbar-tract-1"><i class="fa fa-check"></i><b>13.9.2</b> The Corticobulbar Tract</a></li>
</ul></li>
<li class="chapter" data-level="13.10" data-path="the-somatic-motor-system.html"><a href="the-somatic-motor-system.html#the-extrapyramidal-motor-system"><i class="fa fa-check"></i><b>13.10</b> The Extrapyramidal Motor System</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html"><i class="fa fa-check"></i><b>14</b> Major Diseases Of The Nervous System</a><ul>
<li class="chapter" data-level="14.1" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#neurodegenerative-diseases"><i class="fa fa-check"></i><b>14.1</b> Neurodegenerative Diseases</a><ul>
<li class="chapter" data-level="14.1.1" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#alzheimers-disease"><i class="fa fa-check"></i><b>14.1.1</b> Alzheimer’s disease</a></li>
<li class="chapter" data-level="14.1.2" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#parkinsons-disease"><i class="fa fa-check"></i><b>14.1.2</b> Parkinson’s disease</a></li>
<li class="chapter" data-level="14.1.3" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#huntingtons-disease"><i class="fa fa-check"></i><b>14.1.3</b> Huntington’s disease</a></li>
<li class="chapter" data-level="14.1.4" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#amyotrophic-lateral-sclerosis-als"><i class="fa fa-check"></i><b>14.1.4</b> Amyotrophic lateral sclerosis (ALS)</a></li>
</ul></li>
<li class="chapter" data-level="14.2" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#cerebrovascular-disease"><i class="fa fa-check"></i><b>14.2</b> Cerebrovascular Disease</a><ul>
<li class="chapter" data-level="14.2.1" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#stroke"><i class="fa fa-check"></i><b>14.2.1</b> Stroke</a></li>
<li class="chapter" data-level="14.2.2" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#aphasia"><i class="fa fa-check"></i><b>14.2.2</b> Aphasia</a></li>
<li class="chapter" data-level="14.2.3" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#hemispatial-neglect"><i class="fa fa-check"></i><b>14.2.3</b> Hemispatial neglect</a></li>
</ul></li>
<li class="chapter" data-level="14.3" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#epilepsy"><i class="fa fa-check"></i><b>14.3</b> Epilepsy</a></li>
<li class="chapter" data-level="14.4" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#mental-disorders"><i class="fa fa-check"></i><b>14.4</b> Mental Disorders</a><ul>
<li class="chapter" data-level="14.4.1" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#mood-disorders"><i class="fa fa-check"></i><b>14.4.1</b> Mood Disorders</a></li>
<li class="chapter" data-level="14.4.2" data-path="major-diseases-of-the-nervous-system.html"><a href="major-diseases-of-the-nervous-system.html#schizophrenia"><i class="fa fa-check"></i><b>14.4.2</b> Schizophrenia</a></li>
</ul></li>
</ul></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html"><i class="fa fa-check"></i><b>A</b> Anatomical Terms Of Location</a><ul>
<li class="chapter" data-level="A.1" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#anatomical-planes"><i class="fa fa-check"></i><b>A.1</b> Anatomical Planes</a></li>
<li class="chapter" data-level="A.2" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#anatomical-axes"><i class="fa fa-check"></i><b>A.2</b> Anatomical Axes</a></li>
<li class="chapter" data-level="A.3" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#main-anatomical-terms"><i class="fa fa-check"></i><b>A.3</b> Main Anatomical Terms</a><ul>
<li class="chapter" data-level="A.3.1" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#superior-and-inferior"><i class="fa fa-check"></i><b>A.3.1</b> Superior And Inferior</a></li>
<li class="chapter" data-level="A.3.2" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#anterior-and-posterior"><i class="fa fa-check"></i><b>A.3.2</b> Anterior And Posterior</a></li>
<li class="chapter" data-level="A.3.3" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#medial-and-lateral"><i class="fa fa-check"></i><b>A.3.3</b> Medial And Lateral</a></li>
<li class="chapter" data-level="A.3.4" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#central-and-peripheral"><i class="fa fa-check"></i><b>A.3.4</b> Central And Peripheral</a></li>
<li class="chapter" data-level="A.3.5" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#superficial-and-deep"><i class="fa fa-check"></i><b>A.3.5</b> Superficial And Deep</a></li>
<li class="chapter" data-level="A.3.6" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#dorsal-and-ventral"><i class="fa fa-check"></i><b>A.3.6</b> Dorsal And Ventral</a></li>
<li class="chapter" data-level="A.3.7" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#cranial-and-caudal"><i class="fa fa-check"></i><b>A.3.7</b> Cranial And Caudal</a></li>
</ul></li>
<li class="chapter" data-level="A.4" data-path="anatomical-terms-of-location.html"><a href="anatomical-terms-of-location.html#prefixes"><i class="fa fa-check"></i><b>A.4</b> Prefixes</a></li>
</ul></li>
<li class="chapter" data-level="B" data-path="neuroanatomical-terms.html"><a href="neuroanatomical-terms.html"><i class="fa fa-check"></i><b>B</b> Neuroanatomical Terms</a><ul>
<li class="chapter" data-level="B.1" data-path="neuroanatomical-terms.html"><a href="neuroanatomical-terms.html#nucleus"><i class="fa fa-check"></i><b>B.1</b> Nucleus</a></li>
<li class="chapter" data-level="B.2" data-path="neuroanatomical-terms.html"><a href="neuroanatomical-terms.html#ganglion"><i class="fa fa-check"></i><b>B.2</b> Ganglion</a></li>
<li class="chapter" data-level="B.3" data-path="neuroanatomical-terms.html"><a href="neuroanatomical-terms.html#tract"><i class="fa fa-check"></i><b>B.3</b> Tract</a></li>
<li class="chapter" data-level="B.4" data-path="neuroanatomical-terms.html"><a href="neuroanatomical-terms.html#lemniscus"><i class="fa fa-check"></i><b>B.4</b> Lemniscus</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="list-of-regions-in-the-human-brain.html"><a href="list-of-regions-in-the-human-brain.html"><i class="fa fa-check"></i><b>C</b> List Of Regions In The Human Brain</a><ul>
<li class="chapter" data-level="C.1" data-path="list-of-regions-in-the-human-brain.html"><a href="list-of-regions-in-the-human-brain.html#hindbrain-rhombencephalon"><i class="fa fa-check"></i><b>C.1</b> Hindbrain (rhombencephalon)</a></li>
<li class="chapter" data-level="C.2" data-path="list-of-regions-in-the-human-brain.html"><a href="list-of-regions-in-the-human-brain.html#midbrain-mesencephalon"><i class="fa fa-check"></i><b>C.2</b> Midbrain (mesencephalon)</a></li>
<li class="chapter" data-level="C.3" data-path="list-of-regions-in-the-human-brain.html"><a href="list-of-regions-in-the-human-brain.html#forebrain-prosencephalon"><i class="fa fa-check"></i><b>C.3</b> Forebrain (prosencephalon)</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="neural-pathways.html"><a href="neural-pathways.html"><i class="fa fa-check"></i><b>D</b> Neural pathways</a><ul>
<li class="chapter" data-level="D.1" data-path="neural-pathways.html"><a href="neural-pathways.html#motor-systems-descending-fibers"><i class="fa fa-check"></i><b>D.1</b> Motor systems / Descending fibers</a></li>
<li class="chapter" data-level="D.2" data-path="neural-pathways.html"><a href="neural-pathways.html#somatosensory-system"><i class="fa fa-check"></i><b>D.2</b> Somatosensory system</a></li>
<li class="chapter" data-level="D.3" data-path="neural-pathways.html"><a href="neural-pathways.html#visual-system"><i class="fa fa-check"></i><b>D.3</b> Visual system</a></li>
<li class="chapter" data-level="D.4" data-path="neural-pathways.html"><a href="neural-pathways.html#auditory-system"><i class="fa fa-check"></i><b>D.4</b> Auditory system</a></li>
</ul></li>
<li class="chapter" data-level="E" data-path="brodmann-areas.html"><a href="brodmann-areas.html"><i class="fa fa-check"></i><b>E</b> Brodmann Areas</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Text For BIO400 Neuroanatomy at Salem State University</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="electrical-basis-of-neuronal-function" class="section level1">
<h1><span class="header-section-number">4</span> Electrical Basis Of Neuronal Function</h1>
<div id="the-membrane-potential" class="section level2">
<h2><span class="header-section-number">4.1</span> The Membrane Potential</h2>
<p>Membrane potential (also transmembrane potential or membrane voltage) is the difference in electric potential between the interior and the exterior of a biological cell. With respect to the exterior of the cell, typical values of membrane potential, normally given in units of millivolts and denoted as mV, ranges from –40 mV to –80 mV.</p>
<p>All animal cells are surrounded by a membrane composed of a lipid bilayer with proteins embedded in it. The membrane serves as both an insulator and a diffusion barrier to the movement of ions. Transmembrane proteins, also known as ion transporter or ion pump proteins, actively push ions across the membrane and establish concentration gradients across the membrane, and ion channels allow ions to move across the membrane down those concentration gradients. Ion pumps and ion channels are electrically equivalent to a set of batteries and resistors inserted in the membrane, and therefore create a voltage between the two sides of the membrane.</p>
<p>Almost all plasma membranes have an electrical potential across them, with the inside usually negative with respect to the outside. The membrane potential has two basic functions. First, it allows a cell to function as a battery, providing power to operate a variety of “molecular devices” embedded in the membrane. Second, in electrically excitable cells such as neurons and muscle cells, it is used for transmitting signals between different parts of a cell. Signals are generated by opening or closing of ion channels at one point in the membrane, producing a local change in the membrane potential. This change in the electric field can quickly affect adjacent and more distant ion channels in the membrane. Those ion channels can then open or close as a result of the potential change, reproducing the signal.</p>
<p>In non-excitable cells, and in excitable cells in their baseline states, the membrane potential is held at a relatively stable value, called the resting potential. For neurons, typical values of the resting potential range from –70 to –80 millivolts; that is, the interior of a cell has a negative baseline voltage of a bit less than one-tenth of a volt. The opening and closing of ion channels can induce a departure from the resting potential. This is called a depolarization if the interior voltage becomes less negative (say from –70 mV to –60 mV), or a hyperpolarization if the interior voltage becomes more negative (say from –70 mV to –80 mV). In excitable cells, a sufficiently large depolarization can evoke an action potential, in which the membrane potential changes rapidly and significantly for a short time (on the order of 1 to 100 milliseconds), often reversing its polarity. Action potentials are generated by the activation of certain voltage-gated ion channels.</p>
<p>In neurons, the factors that influence the membrane potential are diverse. They include numerous types of ion channels, some of which are chemically gated and some of which are voltage-gated. Because voltage-gated ion channels are controlled by the membrane potential, while the membrane potential itself is influenced by these same ion channels, feedback loops that allow for complex temporal dynamics arise, including oscillations and regenerative events such as action potentials.</p>
<p>The membrane potential in a cell derives ultimately from two factors: electrical force and diffusion. Electrical force arises from the mutual attraction between particles with opposite electrical charges (positive and negative) and the mutual repulsion between particles with the same type of charge (both positive or both negative). Diffusion arises from the statistical tendency of particles to redistribute from regions where they are highly concentrated to regions where the concentration is low.</p>
<p>Voltage, which is synonymous with difference in electrical potential, is the ability to drive an electric current across a resistance. Indeed, the simplest definition of a voltage is given by Ohm’s law: V=IR, where V is voltage, I is current and R is resistance. If a voltage source such as a battery is placed in an electrical circuit, the higher the voltage of the source the greater the amount of current that it will drive across the available resistance. The functional significance of voltage lies only in potential differences between two points in a circuit. The idea of a voltage at a single point is meaningless. It is conventional in electronics to assign a voltage of zero to some arbitrarily chosen element of the circuit, and then assign voltages for other elements measured relative to that zero point. There is no significance in which element is chosen as the zero point—the function of a circuit depends only on the differences not on voltages per se. However, in most cases and by convention, the zero level is most often assigned to the portion of a circuit that is in contact with ground.</p>
<p>The same principle applies to voltage in cell biology. In electrically active tissue, the potential difference between any two points can be measured by inserting an electrode at each point, for example one inside and one outside the cell, and connecting both electrodes to the leads of what is in essence a specialized voltmeter. By convention, the zero potential value is assigned to the outside of the cell and the sign of the potential difference between the outside and the inside is determined by the potential of the inside relative to the outside zero.</p>
</div>
<div id="ions-and-the-forces-driving-their-motion" class="section level2">
<h2><span class="header-section-number">4.2</span> Ions And The Forces Driving Their Motion</h2>
<p>Electrical signals within biological organisms are, in general, driven by ions. An ion is an atom or molecule that has a net electrical charge. Since the charge of the electron (considered negative by convention) is equal and opposite to that of the proton (considered positive by convention), the net charge of an ion is non-zero due to its total number of electrons being unequal to its total number of protons. A cation is a positively charged ion, with fewer electrons than protons, while an anion is negatively charged, with more electrons than protons. Because of their opposite electric charges, cations and anions attract each other and readily form ionic compounds.</p>
<p>The word ion comes from the Greek word ἰόν, ion, “going”, the present participle of ἰέναι, ienai, “to go”. This term was introduced (after a suggestion by <a href="https://en.wikipedia.org/wiki/William_Whewell">William Whewell</a>) by English physicist and chemist <a href="https://en.wikipedia.org/wiki/Michael_Faraday">Michael Faraday</a> in 1834 for the then-unknown species that goes from one electrode to the other through an aqueous medium. Faraday did not know the nature of these species, but he knew that since metals dissolved into and entered a solution at one electrode and new metal came forth from a solution at the other electrode; that some kind of substance has moved through the solution in a current. This conveys matter from one place to the other. In correspondence with Faraday, Whewell also coined the words anode and cathode, as well as anion (from the Greek word ἄνω (ánō), meaning “up)”and cation (from the Greek word κάτω (káto), meaning “down”) as ions that are attracted to the respective electrodes.</p>
<p><a href="https://en.wikipedia.org/wiki/Svante_Arrhenius">Svante Arrhenius</a> put forth, in his 1884 dissertation, his explanation of the fact that solid crystalline salts dissociate into paired charged particles when dissolved, for which he would win the 1903 Nobel Prize in Chemistry. Arrhenius’ explanation was that in forming a solution, the salt dissociates into Faraday’s ions. Arrhenius proposed that ions formed even in the absence of an electric current.</p>
<p>The most important cations for the action potential are sodium (Na<sup>+</sup>) and potassium (K<sup>+</sup>). Both of these are monovalent cations that carry a single positive charge. Action potentials can also involve calcium (Ca<sup>2+</sup>), which is a divalent cation that carries a double positive charge. The chloride anion (Cl<sup>−</sup>) plays a major role in the action potentials of some algae, but plays a negligible role in the action potentials of most animals.</p>

<div class="figure" style="text-align: center"><span id="fig:ionsize"></span>
<img src="figures/potential/Atomic_and_ionic_radii.svg" alt=": Radii of some elements as neutral atoms, and cations (red) and anions (blue) derived from those atoms. Radii are given in picometers." width="70%" />
<p class="caption">
Figure 4.1: : <a href="https://commons.wikimedia.org/wiki/File:Atomic_%26_ionic_radii.svg">Radii of some elements as neutral atoms, and cations (red) and anions (blue) derived from those atoms. Radii are given in picometers.</a>
</p>
</div>
<p>Ions cross the cell membrane under two influences: diffusion and electric fields. A simple example wherein two solutions—A and B—are separated by a porous barrier illustrates that diffusion will ensure that they will eventually mix into equal solutions. This mixing occurs because of the difference in their concentrations. The region with high concentration will diffuse out toward the region with low concentration. To extend the example, let solution A have 30 sodium ions and 30 chloride ions. Also, let solution B have only 20 sodium ions and 20 chloride ions. Assuming the barrier allows both types of ions to travel through it, then a steady state will be reached whereby both solutions have 25 sodium ions and 25 chloride ions. If, however, the porous barrier is selective to which ions are let through, then diffusion alone will not determine the resulting solution. Returning to the previous example, let’s now construct a barrier that is permeable only to sodium ions. Now, only sodium is allowed to diffuse cross the barrier from its higher concentration in solution A to the lower concentration in solution B. This will result in a greater accumulation of sodium ions than chloride ions in solution B and a lesser number of sodium ions than chloride ions in solution A.</p>
<p>This means that there is a net positive charge in solution B from the higher concentration of positively charged sodium ions than negatively charged chloride ions. Likewise, there is a net negative charge in solution A from the greater concentration of negative chloride ions than positive sodium ions. Since opposite charges attract and like charges repel, the ions are now also influenced by electrical fields as well as forces of diffusion. Therefore, positive sodium ions will be less likely to travel to the now-more-positive B solution and remain in the now-more-negative A solution. The point at which the forces of the electric fields completely counteract the force due to diffusion is called the equilibrium potential. At this point, the net flow of the specific ion (in this case sodium) is zero.</p>
</div>
<div id="the-plasma-membrane" class="section level2">
<h2><span class="header-section-number">4.3</span> The Plasma Membrane</h2>
<p>The cell membrane, also called the plasma membrane or plasmalemma, is a semipermeable lipid bilayer common to all living cells. It contains a variety of biological molecules, primarily proteins and lipids, which are involved in a vast array of cellular processes.</p>

<div class="figure" style="text-align: center"><span id="fig:cellmembrane"></span>
<img src="figures/potential/cell_membrane_rhodopsin_water_Cl_K.png" alt="Picture of a molecular dynamics simulation of a cell membrane/protein complex consisting of bovine rhodopsin incorporated of a phosphatidylcholine (1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine, POPC) lipid bylayer. POPC and water molecules are depicted as sticks. The lipid layers facing the extracellular and cytoplasmic spaces are shown in white and blue, respectively. Both the extra- and intracellular interfaces are covered with layers of water. The secondary structure of rhodopsin is depicted in rainbow colored cartoon representation. Potassium and chloride ions are shown as spheres (colored in cyan and green, respectively). Image generated from PDB file obtained from the CHARMM-GUI Archive - Protein/Membrane Complex Library using the open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 4.2: Picture of a <a href="https://doi.org/10.1371/journal.pone.0000880">molecular dynamics simulation</a> of a cell membrane/protein complex consisting of bovine <a href="https://en.wikipedia.org/wiki/Rhodopsin">rhodopsin</a> incorporated of a phosphatidylcholine (1-palmitoyl-2-oleoyl-sn-glycero-3-phosphocholine, POPC) lipid bylayer. POPC and water molecules are depicted as sticks. The lipid layers facing the extracellular and cytoplasmic spaces are shown in white and blue, respectively. Both the extra- and intracellular interfaces are covered with layers of water. The secondary structure of rhodopsin is depicted in rainbow colored cartoon representation. Potassium and chloride ions are shown as spheres (colored in cyan and green, respectively). Image generated from <a href="http://www.charmm-gui.org/archive/complex/1gzm_rect_popc.pdb">PDB file</a> obtained from the <a href="http://www.charmm-gui.org/?doc=archive&amp;lib=complex">CHARMM-GUI Archive - Protein/Membrane Complex Library</a> using the open source molecular visualization tool <a href="https://pymol.org/2/">PyMol</a>.
</p>
</div>
<p>Every animal cell is enclosed in a plasma membrane, which has the structure of a lipid bilayer with many types of large molecules embedded in it. Because it is made of lipid molecules, the plasma membrane intrinsically has a high electrical resistivity, in other words a low intrinsic permeability to ions. However, some of the molecules embedded in the membrane are capable either of actively transporting ions from one side of the membrane to the other or of providing channels through which they can move.</p>
<p>In electrical terminology, the plasma membrane functions as a combined resistor and capacitor. Resistance arises from the fact that the membrane impedes the movement of charges across it. Capacitance arises from the fact that the lipid bilayer is so thin that an accumulation of charged particles on one side gives rise to an electrical force that pulls oppositely charged particles toward the other side. The capacitance of the membrane is relatively unaffected by the molecules that are embedded in it, so it has a more or less invariant value estimated at about 2 μF/cm<sup>2</sup> (the total capacitance of a patch of membrane is proportional to its area). The conductance of a pure lipid bilayer is so low, on the other hand, that in biological situations it is always dominated by the conductance of alternative pathways provided by embedded molecules. Thus, the capacitance of the membrane is more or less fixed, but the resistance is highly variable.</p>
<p>The thickness of a plasma membrane is estimated to be about 7-8 nanometers. Because the membrane is so thin, it does not take a very large transmembrane voltage to create a strong electric field within it. Typical membrane potentials in animal cells are on the order of 100 millivolts (that is, one tenth of a volt), but calculations show that this generates an electric field close to the maximum that the membrane can sustain—it has been calculated that a voltage difference much larger than 200 millivolts could cause dielectric breakdown, that is, arcing across the membrane.</p>
<p>The resistance of a pure lipid bilayer to the passage of ions across it is very high, but structures embedded in the membrane can greatly enhance ion movement, either actively or passively, via mechanisms called facilitated transport and facilitated diffusion. The two types of structure that play the largest roles are ion channels and ion pumps, both usually formed from assemblages of protein molecules. Ion channels provide passageways through which ions can move. In most cases, an ion channel is permeable only to specific types of ions (for example, sodium and potassium but not chloride or calcium), and sometimes the permeability varies depending on the direction of ion movement. Ion pumps, also known as ion transporters or carrier proteins, actively transport specific types of ions from one side of the membrane to the other, using energy derived from metabolic processes to do so.</p>
</div>
<div id="ion-pumps" class="section level2">
<h2><span class="header-section-number">4.4</span> Ion Pumps</h2>
<p>Ion pumps are integral membrane proteins that carry out active transport, i.e., use cellular energy (ATP) to “pump” the ions against their concentration gradient. Such ion pumps take in ions from one side of the membrane (decreasing its concentration there) and release them on the other side (increasing its concentration there).</p>
<p>The ion pump most relevant to the action potential is the sodium–potassium pump, which transports three sodium ions out of the cell and two potassium ions in. As a consequence, the concentration of potassium ions K<sup>+</sup> inside the neuron is roughly 20-fold larger than the outside concentration, whereas the sodium concentration outside is roughly ninefold larger than inside. In a similar manner, other ions have different concentrations inside and outside the neuron, such as calcium, chloride and magnesium.</p>

<div class="figure" style="text-align: center"><span id="fig:nakpump"></span>
<img src="figures/potential/Na_K_pump.png" alt="A cartoon representation of the molecular structure of the sodium - potassium pump based on atomic coordinates of PDB 2ZXE, rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 4.3: A cartoon representation of the molecular structure of the sodium - potassium pump based on atomic coordinates of <a href="https://www.rcsb.org/structure/2ZXE">PDB 2ZXE</a>, rendered with open source molecular visualization tool PyMol.
</p>
</div>
<p>If the numbers of each type of ion were equal, the sodium–potassium pump would be electrically neutral, but, because of the three-for-two exchange, it gives a net movement of one positive charge from intracellular to extracellular for each cycle, thereby contributing to a positive voltage difference. The pump has three effects: (1) it makes the sodium concentration high in the extracellular space and low in the intracellular space; (2) it makes the potassium concentration high in the intracellular space and low in the extracellular space; (3) it gives the intracellular space a negative voltage with respect to the extracellular space.</p>
<p>The sodium-potassium pump is relatively slow in operation. If a cell were initialized with equal concentrations of sodium and potassium everywhere, it would take hours for the pump to establish equilibrium. The pump operates constantly, but becomes progressively less efficient as the concentrations of sodium and potassium available for pumping are reduced.</p>
<p>Ion pumps influence the action potential only by establishing the relative ratio of intracellular and extracellular ion concentrations. The action potential involves mainly the opening and closing of ion channels not ion pumps. If the ion pumps are turned off by removing their energy source, or by adding an inhibitor such as ouabain, the axon can still fire hundreds of thousands of action potentials before their amplitudes begin to decay significantly. In particular, ion pumps play no significant role in the repolarization of the membrane after an action potential.</p>
<p>Another functionally important ion pump is the sodium-calcium exchanger. This pump operates in a conceptually similar way to the sodium-potassium pump, except that in each cycle it exchanges three Na<sup>+</sup> from the extracellular space for one Ca<sup>2+</sup> from the intracellular space. Because the net flow of charge is inward, this pump runs “downhill”, in effect, and therefore does not require any energy source except the membrane voltage. Its most important effect is to pump calcium outward—it also allows an inward flow of sodium, thereby counteracting the sodium-potassium pump, but, because overall sodium and potassium concentrations are much higher than calcium concentrations, this effect is relatively unimportant. The net result of the sodium-calcium exchanger is that in the resting state, intracellular calcium concentrations become very low.</p>
</div>
<div id="ion-channels" class="section level2">
<h2><span class="header-section-number">4.5</span> Ion Channels</h2>
<p>Ion channels are integral membrane proteins with a pore through which ions can travel between extracellular space and cell interior. Most channels are specific (selective) for one ion; for example, most potassium channels are characterized by 1000:1 selectivity ratio for potassium over sodium, though potassium and sodium ions have the same charge and differ only slightly in their radius. The channel pore is typically so small that ions must pass through it in single-file order. Channel pores can be either open or closed for ion passage, although a number of channels demonstrate various sub-conductance levels. When a channel is open, ions permeate through the channel pore down the transmembrane concentration gradient for that particular ion. Rate of ionic flow through the channel, i.e. single-channel current amplitude, is determined by the maximum channel conductance and electrochemical driving force for that ion, which is the difference between the instantaneous value of the membrane potential and the value of the reversal potential.</p>
<p>The fundamental properties of currents mediated by ion channels were analyzed by the British biophysicists <a href="https://en.wikipedia.org/wiki/Alan_Hodgkin">Alan Hodgkin</a> and <a href="https://en.wikipedia.org/wiki/Andrew_Huxley">Andrew Huxley</a> as part of their Nobel Prize-winning research on the action potential, published in 1952. They built on the work of other physiologists, such as <a href="https://en.wikipedia.org/wiki/Kenneth_Stewart_Cole">Kenneth Stewart Cole’s</a> research into voltage-gated membrane pores from 1941. The existence of ion channels was confirmed in the 1970s by <a href="https://en.wikipedia.org/wiki/Bernard_Katz">Bernard Katz</a> and <a href="https://en.wikipedia.org/wiki/Ricardo_Miledi">Ricardo Miledi</a> using noise analysis. It was then shown more directly with an electrical recording technique known as the “patch clamp”, which led to a Nobel Prize to <a href="https://en.wikipedia.org/wiki/Erwin_Neher">Erwin Neher</a> and <a href="https://en.wikipedia.org/wiki/Bert_Sakmann">Bert Sakmann</a>, the technique’s inventors. Hundreds if not thousands of researchers continue to pursue a more detailed understanding of how these proteins work.</p>
<p>Channels differ with respect to the ion they let pass (for example, Na<sup>+</sup>, K<sup>+</sup>, Cl<sup>−</sup>), the ways in which they may be regulated, the number of subunits of which they are composed and other aspects of structure. Channels belonging to the largest class, which includes the voltage-gated channels that underlie the nerve impulse, consists of four subunits with six transmembrane helices each. On activation, these helices move about and open the pore. Two of these six helices are separated by a loop that lines the pore and is the primary determinant of ion selectivity and conductance in this channel class and some others. The existence and mechanism for ion selectivity was first postulated in the late 1960s by <a href="https://en.wikipedia.org/wiki/Bertil_Hille">Bertil Hille</a> and <a href="https://en.wikipedia.org/wiki/Clay_Armstrong">Clay Armstrong</a>. The idea of the ionic selectivity for potassium channels was that the carbonyl oxygens of the protein backbones of the “selectivity filter” (named by Bertil Hille) could efficiently replace the water molecules that normally shield potassium ions, but that sodium ions were smaller and cannot be completely dehydrated to allow such shielding, and therefore could not pass through. This mechanism was finally confirmed when the first structure of an ion channel was elucidated. A bacterial potassium channel KcsA, consisting of just the selectivity filter, “P” loop and two transmembrane helices was used as a model to study the permeability and the selectivity of ion channels. The determination of the molecular structure of KcsA by <a href="https://en.wikipedia.org/wiki/Roderick_MacKinnon">Roderick MacKinnon</a> using X-ray crystallography won a share of the 2003 Nobel Prize in Chemistry.</p>

<div class="figure" style="text-align: center"><span id="fig:kchannel"></span>
<img src="figures/potential/Kcsa.png" alt="The structure of the potassium channel KcsA fromn Streptomyces lividans determined by X-ray crystallography.Top: side view; bottom: top view. KcsA shares sequence similarity with all known K+ channels and was the first ion channel to have its structure solved at atomic resolution. It consists of four identical subunits that together form a cone shaped structure (top) with a ion selectivity filter at its outer end. Three K+ ions (red spheres) are present in the channel, two of which are held 7.5 angstroms apart by the selectivity filter, a third K+ ion shown in the channel pore below. Data from PDB 1BL8, rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 4.4: The structure of the potassium channel KcsA fromn <em>Streptomyces lividans</em> determined by X-ray crystallography.Top: side view; bottom: top view. KcsA shares sequence similarity with all known K<sup>+</sup> channels and was the first ion channel to have its structure solved at atomic resolution. It consists of four identical subunits that together form a cone shaped structure (top) with a ion selectivity filter at its outer end. Three K<sup>+</sup> ions (red spheres) are present in the channel, two of which are held 7.5 angstroms apart by the selectivity filter, a third K<sup>+</sup> ion shown in the channel pore below. Data from <a href="https://www.rcsb.org/structure/1BL8">PDB 1BL8</a>, rendered with open source molecular visualization tool <a href="https://pymol.org/2/">PyMol</a>.
</p>
</div>
<p>Because of their small size and the difficulty of crystallizing integral membrane proteins for X-ray analysis, it is only very recently that scientists have been able to directly examine what channels “look like.” Most of what researchers have deduced about channel operation so far they have established through electrophysiology, biochemistry, gene sequence comparison and mutagenesisi and structural studies (X-ray crystallography and cryoelectronmicroscopy).</p>
<p>Channels can have single (e.g. members of the Chloride Intracellular Ion Channel family) or multiple transmembrane (K<sup>+</sup> channels, P2X receptors, Na<sup>+</sup> channels) domains which span the plasma membrane to form pores.</p>
<p>Ion channels can be classified by how they respond to their environment. For example, the ion channels involved in the action potential are voltage-sensitive channels; they open and close in response to the voltage across the membrane. Ligand-gated channels form another important class; these ion channels open and close in response to the binding of a ligand molecule, such as a neurotransmitter. Other ion channels open and close with mechanical forces. Still other ion channels—such as those of sensory neurons—open and close in response to other stimuli, such as light, temperature or pressure.</p>
<p>Leakage channels are the simplest type of ion channel, in that their permeability is more or less constant. The types of leakage channels that have the greatest significance in neurons are potassium and chloride channels. Even these are not perfectly constant in their properties: First, most of them are voltage-dependent in the sense that they conduct better in one direction than the other (in other words, they are rectifiers); second, some of them are capable of being shut off by chemical ligands even though they do not require ligands in order to operate.</p>
<p><a href="https://en.wikipedia.org/wiki/Voltage-gated_ion_channel">Voltage-gated ion channels</a> are a class of transmembrane proteins that form ion channels that are activated by changes in the electrical membrane potential near the channel. The membrane potential alters the conformation of the channel proteins, regulating their opening and closing. Cell membranes are generally impermeable to ions, thus they must diffuse through the membrane through transmembrane protein channels. They have a crucial role in excitable cells such as neuronal and muscle tissues, allowing a rapid and co-ordinated depolarization in response to triggering voltage change. Found along the axon and at the synapse, voltage-gated ion channels directionally propagate electrical signals. Voltage-gated ion-channels are usually ion-specific, and channels specific to sodium (Na<sup>+</sup>), potassium (K<sup>+</sup>), calcium (Ca<sup>2+</sup>), and chloride (Cl−) ions have been identified. The opening and closing of the channels are triggered by changing ion concentration, and hence charge gradient, between the sides of the cell membrane.</p>
<p>Voltage-gated ion channels are generally composed of several subunits arranged in such a way that there is a central pore through which ions can travel down their electrochemical gradients. The channels tend to be ion-specific, although similarly sized and charged ions may sometimes travel through them. The functionality of voltage-gated ion channels is attributed to its three main discrete units: the voltage sensor, the pore or conducting pathway, and the gate. Na<sup>+</sup>, K<sup>+</sup>, and Ca<sup>2+</sup> channels are composed of four transmembrane domains arranged around a central pore; these four domains are part of a single α-subunit in the case of most Na<sup>+</sup> and Ca<sup>2+</sup> channels, whereas there are four α-subunits, each contributing one transmembrane domain, in most K<sup>+</sup> channels. The membrane-spanning segments, designated S1-S6, all take the form of alpha helices with specialized functions. The fifth and sixth transmembrane segments (S5 and S6) and pore loop serve the principal role of ion conduction, comprising the gate and pore of the channel, while S1-S4 serve as the voltage-sensing region. The four subunits may be identical, or different from one another. In addition to the four central α-subunits, there are also regulatory β-subunits, with oxidoreductase activity, which are located on the inner surface of the cell membrane and do not cross the membrane, and which are coassembled with the α-subunits in the endoplasmic reticulum.</p>
<p>Crystallographic structural studies of a potassium channel have shown that, when a potential difference is introduced over the membrane, the associated electric field induces a conformational change in the potassium channel. The conformational change distorts the shape of the channel proteins sufficiently such that the cavity, or channel, opens to allow influx or efflux to occur across the membrane. This movement of ions down their concentration gradients subsequently generates an electric current sufficient to depolarize the cell membrane.</p>
<p>Voltage-gated sodium channels and calcium channels are made up of a single polypeptide with four homologous domains. Each domain contains 6 membrane spanning alpha helices. One of these helices, S4, is the voltage sensing helix. The S4 segment contains many positive charges such that a high positive charge outside the cell repels the helix, keeping the channel in its closed state.</p>
<p>In general, the voltage sensing portion of the ion channel is responsible for the detection of changes in transmembrane potential that trigger the opening or closing of the channel. The S1-4 alpha helices are generally thought to serve this role. In potassium and sodium channels, voltage-sensing S4 helices contain positively-charged lysine or arginine residues in repeated motifs. In its resting state, half of each S4 helix is in contact with the cell cytosol. Upon depolarization, the positively-charged residues on the S4 domains move toward the exoplasmic surface of the membrane. It is thought that the first 4 arginines account for the gating current, moving toward the extracellular solvent upon channel activation in response to membrane depolarization. The movement of 10–12 of these protein-bound positive charges triggers a conformational change that opens the channel. The exact mechanism by which this movement occurs is not currently agreed upon, however the canonical, transporter, paddle, and twisted models are examples of current theories.</p>
<p>Movement of the voltage-sensor triggers a conformational change of the gate of the conducting pathway, controlling the flow of ions through the channel.</p>
<p>The main functional part of the voltage-sensitive protein domain of these channels generally contains a region composed of S3b and S4 helices, known as the “paddle” due to its shape, which appears to be a conserved sequence, interchangeable across a wide variety of cells and species. A similar voltage sensor paddle has also been found in a family of voltage sensitive phosphatases in various species. Genetic engineering of the paddle region from a species of volcano-dwelling archaebacteria into rat brain potassium channels results in a fully functional ion channel, as long as the whole intact paddle is replaced. This “modularity” allows use of simple and inexpensive model systems to study the function of this region, its role in disease, and pharmaceutical control of its behavior rather than being limited to poorly characterized, expensive, and/or difficult to study preparations.</p>
<p>Although voltage-gated ion channels are typically activated by membrane depolarization, some channels, such as inward-rectifier potassium ion channels, are activated instead by hyperpolarization.</p>
<p>The gate is thought to be coupled to the voltage sensing regions of the channels and appears to contain a mechanical obstruction to ion flow. While the S6 domain has been agreed upon as the segment acting as this obstruction, its exact mechanism is unknown. Possible explanations include: the S6 segment makes a scissor-like movement allowing ions to flow through, the S6 segment breaks into two segments allowing of passing of ions through the channel, or the S6 channel serving as the gate itself. The mechanism by which the movement of the S4 segment affects that of S6 is still unknown, however it is theorized that there is a S4-S5 linker whose movement allows the opening of S6.</p>
<p>Inactivation of ion channels occurs within milliseconds after opening. Inactivation is thought to be mediated by an intracellular gate that controls the opening of the pore on the inside of the cell. This gate is modeled as a ball tethered to a flexible chain. During inactivation, the chain folds in on itself and the ball blocks the flow of ions through the channel. Fast inactivation is directly linked to the activation caused by intramembrane movements of the S4 segments, though the mechanism linking movement of S4 and the engagement of the inactivation gate is unknown.</p>
<p>Important representative families of voltage-gated ion channels are the:</p>
<ul>
<li>Voltage-gated sodium channels: This family contains at least 9 members and is largely responsible for action potential creation and propagation. The pore-forming α subunits are very large (up to 4,000 amino acids) and consist of four homologous repeat domains (I-IV) each comprising six transmembrane segments (S1-S6) for a total of 24 transmembrane segments. The members of this family also coassemble with auxiliary β subunits, each spanning the membrane once. Both α and β subunits are extensively glycosylated.</li>
<li>Voltage-gated calcium channels: This family consists of channels that are formed as a complex of several different subunits: α1, α2δ, β1-4, and γ. The α1 subunit forms the Ca<sup>2+</sup> selective ion conducting pore while the associated subunits have several functions including modulation of gating.These channels play an important role in both linking muscle excitation with contraction as well as neuronal excitation with transmitter release. The α subunits have an overall structural resemblance to those of the sodium channels and are equally large.</li>
<li>Cation channels of sperm: This small family of channels, normally referred to as Catsper channels, is related to the two-pore channels and distantly related to TRP channels.</li>
<li>Voltage-gated potassium channels (KV): This family contains almost 40 members, which are further divided into 12 subfamilies. These channels are known mainly for their role in repolarizing the cell membrane following action potentials. The α subunits have six transmembrane segments, homologous to a single domain of the sodium channels. Correspondingly, they assemble as tetramers to produce a functioning channel.</li>
<li>Some transient receptor potential channels: This group of channels, normally referred to simply as TRP channels, is named after their role in <em>Drosophila</em> phototransduction. This family, containing at least 28 members, is incredibly diverse in its method of activation. Some TRP channels seem to be constitutively open, while others are gated by voltage, intracellular Ca<sup>2+</sup>, pH, redox state, osmolarity, and mechanical stretch. These channels also vary according to the ion(s) they pass, some being selective for Ca<sup>2+</sup> while others are less selective, acting as cation channels. This family is subdivided into 6 subfamilies based on homology: classical (TRPC), vanilloid receptors (TRPV), melastatin (TRPM), polycystins (TRPP), mucolipins (TRPML), and ankyrin transmembrane protein 1 (TRPA).</li>
<li>Hyperpolarization-activated cyclic nucleotide-gated channels: The opening of these channels is due to hyperpolarization rather than the depolarization required for other cyclic nucleotide-gated channels. These channels are also sensitive to the cyclic nucleotides cAMP and cGMP, which alter the voltage sensitivity of the channel’s opening. These channels are permeable to the monovalent cations K<sup>+</sup> and Na<sup>+</sup>. There are 4 members of this family, all of which form tetramers of six-transmembrane α subunits. As these channels open under hyperpolarizing conditions, they function as pacemaking channels in the heart, particularly the SA node.</li>
<li>Voltage-gated proton channels: Voltage-gated proton channels open with depolarization, but in a strongly pH-sensitive manner. The result is that these channels open only when the electrochemical gradient is outward, such that their opening will only allow protons to leave cells. Their function thus appears to be acid extrusion from cells. Another important function occurs in phagocytes (e.g. eosinophils, neutrophils, macrophages) during the “respiratory burst.” When bacteria or other microbes are engulfed by phagocytes, the enzyme NADPH oxidase assembles in the membrane and begins to produce reactive oxygen species (ROS) that help kill bacteria. NADPH oxidase is electrogenic, moving electrons across the membrane, and proton channels open to allow proton flux to balance the electron movement electrically.</li>
</ul>

<div class="figure" style="text-align: center"><span id="fig:ionchannel"></span>
<img src="figures/potential/voltage_gated_ion_channels.png" alt="The structures (from left to right) of voltage-gated sodium, potassium, calcium and chloride channels. Top: side view; bottom: top view. Data from PDB 5EK0, PDB 2A79, PDB 3JBR and PDB 1KPL rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 4.5: The structures (from left to right) of voltage-gated sodium, potassium, calcium and chloride channels. Top: side view; bottom: top view. Data from <a href="https://www.rcsb.org/structure/5EK0">PDB 5EK0</a>, <a href="https://www.rcsb.org/structure/2A79">PDB 2A79</a>, <a href="https://www.rcsb.org/structure/3JBR">PDB 3JBR</a> and <a href="https://www.rcsb.org/structure/1KPL">PDB 1KPL</a> rendered with open source molecular visualization tool <a href="https://pymol.org/2/">PyMol</a>.
</p>
</div>
<p><a href="https://en.wikipedia.org/wiki/Ligand-gated_ion_channel">Ligand-gated ion channels</a>, also commonly referred to as ionotropic receptors, are a group of transmembrane ion-channel proteins which open to allow ions such as Na<sup>+</sup>, K<sup>+</sup>, Ca<sup>2+</sup>, and Cl<sup>−</sup> to pass through the membrane in response to the binding of a chemical messenger (i.e. a ligand), such as a neurotransmitter.</p>
<p>When a presynaptic neuron is excited, it releases a neurotransmitter from vesicles into the synaptic cleft. The neurotransmitter then binds to receptors located on the postsynaptic neuron. If these receptors are ligand-gated ion channels, a resulting conformational change opens the ion channels, which leads to a flow of ions across the cell membrane. This, in turn, results in either a depolarization, for an excitatory receptor response, or a hyperpolarization, for an inhibitory response.</p>
<p>These receptor proteins are typically composed of at least two different domains: a transmembrane domain which includes the ion pore, and an extracellular domain which includes the ligand binding location (an allosteric binding site). This modularity has enabled a ‘divide and conquer’ approach to finding the structure of the proteins (crystallising each domain separately). The function of such receptors located at synapses is to convert the chemical signal of presynaptically released neurotransmitter directly and very quickly into a postsynaptic electrical signal. Many ligand-gated ion channels are additionally modulated by allosteric ligands, by channel blockers, ions, or the membrane potential. Ligand-gated ion channels are classified into three superfamilies which lack evolutionary relationship: cys-loop receptors, ionotropic glutamate receptors and ATP-gated channels.</p>
<p>The cys-loop receptors are named after a characteristic loop formed by a disulfide bond between two cysteine residues in the N terminal extracellular domain. They are part of a larger family of pentameric ligand-gated ion channels that usually lack this disulfide bond, hence the tentative name “Pro-loop receptors”. A binding site in the extracellular N-terminal ligand-binding domain gives them receptor specificity for (1) acetylcholine (AcCh), (2) serotonin, (3) glycine, (4) glutamate and (5) γ-aminobutyric acid (GABA) in vertebrates. The receptors are subdivided with respect to the type of ion that they conduct (anionic or cationic) and further into families defined by the endogenous ligand. They are usually pentameric with each subunit containing 4 transmembrane helices constituting the transmembrane domain, and a beta sheet sandwich type, extracellular, N terminal, ligand binding domain. Some also contain an intracellular domain like shown in the image.</p>
<p>The prototypic ligand-gated ion channel is the nicotinic acetylcholine receptor. It consists of a pentamer of protein subunits (typically ααβγδ), with two binding sites for acetylcholine (one at the interface of each alpha subunit). When the acetylcholine binds it alters the receptor’s configuration (twists the T2 helices which moves the leucine residues, which block the pore, out of the channel pathway) and causes the constriction in the pore of approximately 3 angstroms to widen to approximately 8 angstroms so that ions can pass through. This pore allows Na<sup>+</sup> ions to flow down their electrochemical gradient into the cell. With a sufficient number of channels opening at once, the inward flow of positive charges carried by Na<sup>+</sup> ions depolarizes the postsynaptic membrane sufficiently to initiate an action potential.</p>
<p>While single-cell organisms like bacteria would have little apparent need for the transmission of an action potential, a bacterial homologue to an LIC has been identified, hypothesized to act nonetheless as a chemoreceptor. This prokaryotic nAChR variant is known as the GLIC receptor, after the species in which it was identified; Gloeobacter Ligand-gated Ion C channel.</p>
<p>The ionotropic glutamate receptors bind the neurotransmitter glutamate. They form tetramers with each subunit consisting of an extracellular amino terminal domain (ATD, which is involved tetramer assembly), an extracellular ligand binding domain (LBD, which binds glutamate), and a transmembrane domain (TMD, which forms the ion channel). The transmembrane domain of each subunit contains three transmembrane helices as well as a half membrane helix with a reentrant loop. The structure of the protein starts with the ATD at the N terminus followed by the first half of the LBD which is interrupted by helices 1,2 and 3 of the TMD before continuing with the final half of the LBD and then finishing with helix 4 of the TMD at the C terminus. This means there are three links between the TMD and the extracellular domains. Each subunit of the tetramer has a binding site for glutamate formed by the two LBD sections forming a clamshell like shape. Only two of these sites in the tetramer need to be occupied to open the ion channel. The pore is mainly formed by the half helix 2 in a way which resembles an inverted potassium channel.</p>

<div class="figure" style="text-align: center"><span id="fig:lgic"></span>
<img src="figures/potential/ligand_gated_ion_channels.png" alt="Structures of ligand-gated ion channels. Top row: The nicotinic acetylcholine receptor (left) and the serotonin 5-HT3 receptor. Middle row: The GABAA receptor (left) and glycine receptor (rigt). Bottom row: The AMPA receptor (left) and NMDA receptor (reight). Data from PDB 2BG9, PDB 6HIO, PDB 6D6U, PDB 3JAD, PDB 5IDE, and PDB 4PE5, rendered with open source molecular visualization tool PyMol." width="70%" />
<p class="caption">
Figure 4.6: Structures of ligand-gated ion channels. Top row: The nicotinic acetylcholine receptor (left) and the serotonin 5-HT3 receptor. Middle row: The GABA<sub>A</sub> receptor (left) and glycine receptor (rigt). Bottom row: The AMPA receptor (left) and NMDA receptor (reight). Data from <a href="https://www.rcsb.org/structure/2BG9">PDB 2BG9</a>, <a href="https://www.rcsb.org/structure/6HIO">PDB 6HIO</a>, <a href="https://www.rcsb.org/structure/6D6U">PDB 6D6U</a>, <a href="https://www.rcsb.org/structure/3JAD">PDB 3JAD</a>, <a href="https://www.rcsb.org/structure/5IDE">PDB 5IDE</a>, and <a href="https://www.rcsb.org/structure/4PE5">PDB 4PE5</a>, rendered with open source molecular visualization tool <a href="https://pymol.org/2/">PyMol</a>.
</p>
</div>
<p>Also called G protein-coupled receptor, seven-transmembrane domain receptor, 7 TM receptor, constitute a large protein family of receptors that sense molecules outside the cell and activate inside signal transduction pathways and, ultimately, cellular responses. They pass through the cell membrane 7 times. G-protein-Linked receptors are a huge family that have hundreds of members identified. Ion-channel-linked receptors (e.g. GABAB, NMDA, etc.) are only a part of them.</p>
</div>
<div id="the-reversal-potential" class="section level2">
<h2><span class="header-section-number">4.6</span> The Reversal Potential</h2>
<p>The reversal potential (or equilibrium potential) of an ion is the value of transmembrane voltage at which diffusive and electrical forces counterbalance, so that there is no net ion flow across the membrane. This means that the transmembrane voltage exactly opposes the force of diffusion of the ion, such that the net current of the ion across the membrane is zero and unchanging. The reversal potential is important because it corresponds to the voltage that acts on channels permeable to that ion—in other words, it gives the voltage that the ion concentration gradient generates when it acts as a battery.</p>
<p>The equilibrium potential of a particular ion is usually designated by the notation E<sub>ion</sub>.The equilibrium potential for any ion can be calculated using the Nernst equation. For example, reversal potential for potassium ions will be as follows:</p>
<p><span class="math display">\[  E_{eq,K^+} = \frac{RT}{zF} \ln \frac{[K^+]_{o}}{[K^+]_{i}} \]</span></p>
<p>where</p>
<ul>
<li>E<sub>eq,K<sup>+</sup></sub> is the equilibrium potential for potassium, measured in volts</li>
<li>R is the universal gas constant, equal to 8.314 Joule·K<sup>−1</sup>·mol<sup>−1</sup></li>
<li>T is the absolute temperature, measured in Kelvin</li>
<li>z is the number of elementary charges of the ion in question involved in the reaction</li>
<li>F is the Faraday constant, equal to 96,485 Coulomb·mol<sup>−1</sup> or J·V<sup>−1</sup>·mol<sup>−1</sup></li>
<li>[K<sup>+</sup>]<sub>o</sub> is the extracellular concentration of potassium, measured in mol·m<sup>−3</sup> or mmol·l<sup>−1</sup></li>
<li>[K<sup>+</sup>]<sub>i</sub> is the intracellular concentration of potassium</li>
</ul>
<p>Even if two different ions have the same charge (i.e., K<sup>+</sup> and Na<sup>+</sup>), they can still have very different equilibrium potentials, provided their outside and/or inside concentrations differ. Take, for example, the equilibrium potentials of potassium and sodium in neurons. The potassium equilibrium potential E<sub>K</sub> is −84 mV with 5 mM potassium outside and 140 mM inside. On the other hand, the sodium equilibrium potential, E<sub>Na</sub>, is approximately +66 mV with approximately 12 mM sodium inside and 140 mM outside.</p>
<p>A neuron’s resting membrane potential actually changes during the development of an organism. In order for a neuron to eventually adopt its full adult function, its potential must be tightly regulated during development. As an organism progresses through development the resting membrane potential becomes more negative. Glial cells are also differentiating and proliferating as development progresses in the brain. The addition of these glial cells increases the organism’s ability to regulate extracellular potassium. The drop in extracellular potassium can lead to a decrease in membrane potential of 35 mV.</p>
<p>Cell excitability is the change in membrane potential that is necessary for cellular responses in various tissues. Cell excitability is a property that is induced during early embriogenesis. Excitability of a cell has also been defined as the ease with which a response may be triggered. The resting potential forms the basis of cell excitability and these processes are fundamental for the generation of graded and action potentials.</p>
<p>The most important regulators of cell excitability are the extracellular calcium concentration and the calcium-sensing receptor. Calcium is also the most important second messenger in excitable cell signaling. Other important proteins that regulate cell excitability are voltage-gated ion channels, ion transporters, membrane receptors and hyperpolarization-activated cyclic-nucleotide-gated channels. For example, potassium channels are important regulators of excitability in neurons, cardiac myocytes and many other excitable cells like astrocytes. Activation of synaptic receptors initiates long-lasting changes in neuronal excitability.</p>
<p>Many cell types are considered to have an excitable membrane. Excitable cells are neurons, cardiac myocytes, skeletal myocytes, smooth muscle cells, many types of endothelial cells (e.g. beta cells), glial cells (e.g. astrocytes), mechanoreceptor cells (e.g. hair cells and Merkel cells), chemoreceptor cells (e.g. glomus cells, taste receptors), some plant cells and possibly immune cells. Astrocytes display a form of non-electrical excitability based on intracellular calcium variations related to the expression of several receptors through which they can detect the synaptic signal. In neurons, there are different membrane properties in some portions of the cell, for example, dendritic excitability endows neurons with the capacity for coincidence detection of spatially separated inputs.</p>
</div>
<div id="the-resting-potential" class="section level2">
<h2><span class="header-section-number">4.7</span> The Resting Potential</h2>
<p>When the membrane potential of a cell goes for a long period of time without changing significantly, it is referred to as a resting potential or resting voltage. This term is used for the membrane potential of non-excitable cells, but also for the membrane potential of excitable cells in the absence of excitation. In excitable cells, the other possible states are graded membrane potentials (of variable amplitude), and action potentials, which are large, all-or-nothing rises in membrane potential that usually follow a fixed time course. Excitable cells include neurons, muscle cells, and some secretory cells in glands. Even in other types of cells, however, the membrane voltage can undergo changes in response to environmental or intracellular stimuli. For example, depolarization of the plasma membrane appears to be an important step in programmed cell death.</p>
<p>The interactions that generate the resting potential are modeled by the Goldman equation. This is similar in form to the Nernst equation shown above, in that it is based on the charges of the ions in question, as well as the difference between their inside and outside concentrations. However, it also takes into consideration the relative permeability of the plasma membrane to each ion in question.</p>
<p><span class="math display">\[ E_{m} = \frac{RT}{F} \ln{ \left( \frac{ P_{\mathrm{K}}[\mathrm{K}^{+}]_\mathrm{out} + P_{\mathrm{Na}}[\mathrm{Na}^{+}]_\mathrm{out} + P_{\mathrm{Cl}}[\mathrm{Cl}^{-}]_\mathrm{in}}{ P_{\mathrm{K}}[\mathrm{K}^{+}]_\mathrm{in} + P_{\mathrm{Na}}[\mathrm{Na}^{+}]_\mathrm{in} + P_{\mathrm{Cl}}[\mathrm{Cl}^{-}]_\mathrm{out}} \right) } \]</span></p>
<p>The three ions that appear in this equation are potassium (K<sup>+</sup>), sodium (Na<sup>+</sup>), and chloride (Cl<sup>−</sup>). Calcium is omitted, but can be added to deal with situations in which it plays a significant role. Being an anion, the chloride terms are treated differently from the cation terms; the intracellular concentration is in the numerator, and the extracellular concentration in the denominator, which is reversed from the cation terms. <em>P<sub>i</sub></em> stands for the relative permeability of the ion type <em>i</em>.</p>
<p>In essence, the Goldman formula expresses the membrane potential as a weighted average of the reversal potentials for the individual ion types, weighted by permeability. (Although the membrane potential changes about 100 mV during an action potential, the concentrations of ions inside and outside the cell do not change significantly. They remain close to their respective concentrations when the membrane is at resting potential.) In most animal cells, the permeability to potassium is much higher in the resting state than the permeability to sodium. As a consequence, the resting potential is usually close to the potassium reversal potential. The permeability to chloride can be high enough to be significant, but, unlike the other ions, chloride is not actively pumped, and therefore equilibrates at a reversal potential very close to the resting potential determined by the other ions.</p>
<p>Values of resting membrane potential in most animal cells usually vary between the potassium reversal potential (usually around -80 mV) and around -40 mV. The resting potential in excitable cells (capable of producing action potentials) is usually near -60 mV—more depolarized voltages would lead to spontaneous generation of action potentials. Immature or undifferentiated cells show highly variable values of resting voltage, usually significantly more positive than in differentiated cells. In such cells, the resting potential value correlates with the degree of differentiation: undifferentiated cells in some cases may not show any transmembrane voltage difference at all.</p>
<p>Maintenance of the resting potential can be metabolically costly for a cell because of its requirement for active pumping of ions to counteract losses due to leakage channels. The cost is highest when the cell function requires an especially depolarized value of membrane voltage. For example, the resting potential in daylight-adapted blowfly (<em>Calliphora vicina</em>) photoreceptors can be as high as -30 mV. This elevated membrane potential allows the cells to respond very rapidly to visual inputs; the cost is that maintenance of the resting potential may consume more than 20% of overall cellular ATP.</p>
<p>On the other hand, the high resting potential in undifferentiated cells can be a metabolic advantage. This apparent paradox is resolved by examination of the origin of that resting potential. Little-differentiated cells are characterized by extremely high input resistance, which implies that few leakage channels are present at this stage of cell life. As an apparent result, potassium permeability becomes similar to that for sodium ions, which places resting potential in-between the reversal potentials for sodium and potassium as discussed above. The reduced leakage currents also mean there is little need for active pumping in order to compensate, therefore low metabolic cost.</p>
</div>
<div id="the-action-potential" class="section level2">
<h2><span class="header-section-number">4.8</span> The Action Potential</h2>
<p>An action potential occurs when the membrane potential of a specific cell location rapidly rises and falls: this depolarisation then causes adjacent locations to similarly depolarise. Action potentials occur in several types of animal cells, called excitable cells, which include neurons, muscle cells, endocrine cells, glomus cells (peripheral chemoreceptor cells mainly located in the carotid and aortic bodies), and in some plant cells.</p>
<p>In neurons, action potentials play a central role in cell-to-cell communication by providing for—or with regard to saltatory conduction, assisting—the propagation of signals along the neuron’s axon toward synaptic boutons situated at the ends of an axon; these signals can then connect with other neurons at synapses, or to motor cells or glands. In other types of cells, their main function is to activate intracellular processes. In muscle cells, for example, an action potential is the first step in the chain of events leading to contraction. In beta cells of the pancreas, they provoke release of insulin. Action potentials in neurons are also known as “nerve impulses” or “spikes”, and the temporal sequence of action potentials generated by a neuron is called its “spike train”. A neuron that emits an action potential, or nerve impulse, is often said to “fire”.</p>
<p>Action potentials are generated by special types of voltage-gated ion channels embedded in a cell’s plasma membrane. These channels are shut when the membrane potential is near the (negative) resting potential of the cell, but they rapidly begin to open if the membrane potential increases to a precisely defined threshold voltage, depolarising the transmembrane potential. When the channels open, they allow an inward flow of sodium ions, which changes the electrochemical gradient, which in turn produces a further rise in the membrane potential. This then causes more channels to open, producing a greater electric current across the cell membrane and so on. The process proceeds explosively until all of the available ion channels are open, resulting in a large upswing in the membrane potential. The rapid influx of sodium ions causes the polarity of the plasma membrane to reverse, and the ion channels then rapidly inactivate. As the sodium channels close, sodium ions can no longer enter the neuron, and they are then actively transported back out of the plasma membrane. Potassium channels are then activated, and there is an outward current of potassium ions, returning the electrochemical gradient to the resting state. After an action potential has occurred, there is a transient negative shift, called the afterhyperpolarization.</p>
<p>In animal cells, there are two primary types of action potentials. One type is generated by voltage-gated sodium channels, the other by voltage-gated calcium channels. Sodium-based action potentials usually last for under one millisecond, but calcium-based action potentials may last for 100 milliseconds or longer. In some types of neurons, slow calcium spikes provide the driving force for a long burst of rapidly emitted sodium spikes. In cardiac muscle cells, on the other hand, an initial fast sodium spike provides a “primer” to provoke the rapid onset of a calcium spike, which then produces muscle contraction</p>

<div class="figure" style="text-align: center"><span id="fig:actionpotential"></span>
<img src="figures/potential/ActionPotential.svg" alt="Ion movement during an action potential. Key: a) Sodium (Na+) ion. b) Potassium (K+) ion. c) Sodium channel. d) Potassium channel. e) Sodium-potassium pump. In the stages of an action potential, the permeability of the membrane of the neuron changes. At the resting state (1), sodium and potassium ions have limited ability to pass through the membrane, and the neuron has a net negative charge inside. Once the action potential is triggered, the depolarization (2) of the neuron activates sodium channels, allowing sodium ions to pass through the cell membrane into the cell, resulting in a net positive charge in the neuron relative to the extracellular fluid. After the action potential peak is reached, the neuron begins repolarization (3), where the sodium channels close and potassium channels open, allowing potassium ions to cross the membrane into the extracellular fluid, returning the membrane potential to a negative value. Finally, there is a refractory period (4), during which the voltage-dependent ion channels are inactivated while the Na+ and K+ ions return to their resting state distributions across the membrane (1), and the neuron is ready to repeat the process for the next action potential." width="70%" />
<p class="caption">
Figure 4.7: <a href="https://commons.wikimedia.org/wiki/File:Membrane_Permeability_of_a_Neuron_During_an_Action_Potential.svg">Ion movement during an action potential.</a> Key: a) Sodium (Na<sup>+</sup>) ion. b) Potassium (K<sup>+</sup>) ion. c) Sodium channel. d) Potassium channel. e) Sodium-potassium pump. In the stages of an action potential, the permeability of the membrane of the neuron changes. At the resting state (1), sodium and potassium ions have limited ability to pass through the membrane, and the neuron has a net negative charge inside. Once the action potential is triggered, the depolarization (2) of the neuron activates sodium channels, allowing sodium ions to pass through the cell membrane into the cell, resulting in a net positive charge in the neuron relative to the extracellular fluid. After the action potential peak is reached, the neuron begins repolarization (3), where the sodium channels close and potassium channels open, allowing potassium ions to cross the membrane into the extracellular fluid, returning the membrane potential to a negative value. Finally, there is a refractory period (4), during which the voltage-dependent ion channels are inactivated while the Na<sup>+</sup> and K<sup>+</sup> ions return to their resting state distributions across the membrane (1), and the neuron is ready to repeat the process for the next action potential.
</p>
</div>
</div>
<div id="graded-potentials" class="section level2">
<h2><span class="header-section-number">4.9</span> Graded Potentials</h2>
<p>As explained above, the potential at any point in a cell’s membrane is determined by the ion concentration differences between the intracellular and extracellular areas, and by the permeability of the membrane to each type of ion. The ion concentrations do not normally change very quickly (with the exception of Ca<sup>2+</sup>, where the baseline intracellular concentration is so low that even a small influx may increase it by orders of magnitude), but the permeabilities of the ions can change in a fraction of a millisecond, as a result of activation of ligand-gated ion channels. The change in membrane potential can be either large or small, depending on how many ion channels are activated and what type they are, and can be either long or short, depending on the lengths of time that the channels remain open. Changes of this type are referred to as graded potentials, in contrast to action potentials, which have a fixed amplitude and time course.</p>
<p>As can be derived from the Goldman equation shown above, the effect of increasing the permeability of a membrane to a particular type of ion shifts the membrane potential toward the reversal potential for that ion. Thus, opening Na<sup>+</sup> channels shifts the membrane potential toward the Na<sup>+</sup> reversal potential, which is usually around +100 mV. Likewise, opening K<sup>+</sup> channels shifts the membrane potential toward about –90 mV, and opening Cl<sup>−</sup> channels shifts it toward about –70 mV (resting potential of most membranes). Thus, Na<sup>+</sup> channels shift the membrane potential in a positive direction, K<sup>+</sup> channels shift it in a negative direction (except when the membrane is hyperpolarized to a value more negative than the K<sup>+</sup> reversal potential), and Cl<sup>−</sup> channels tend to shift it towards the resting potential.</p>
<p>Graded membrane potentials are particularly important in neurons, where they are produced by synapses—a temporary change in membrane potential produced by activation of a synapse by a single graded or action potential is called a postsynaptic potential. Neurotransmitters that act to open Na<sup>+</sup> channels typically cause the membrane potential to become more positive, while neurotransmitters that activate K<sup>+</sup> channels typically cause it to become more negative; those that inhibit these channels tend to have the opposite effect.</p>
<p>Whether a postsynaptic potential is considered excitatory or inhibitory depends on the reversal potential for the ions of that current, and the threshold for the cell to fire an action potential (around –50mV). A postsynaptic current with a reversal potential above threshold, such as a typical Na<sup>+</sup> current, is considered excitatory. A current with a reversal potential below threshold, such as a typical K<sup>+</sup> current, is considered inhibitory. A current with a reversal potential above the resting potential, but below threshold, will not by itself elicit action potentials, but will produce subthreshold membrane potential oscillations. Thus, neurotransmitters that act to open Na<sup>+</sup> channels produce excitatory postsynaptic potentials, or EPSPs, whereas neurotransmitters that act to open K<sup>+</sup> or Cl<sup>−</sup> channels typically produce inhibitory postsynaptic potentials, or IPSPs. When multiple types of channels are open within the same time period, their postsynaptic potentials summate (are added together).</p>
<p>From the viewpoint of biophysics, the resting membrane potential is merely the membrane potential that results from the membrane permeabilities that predominate when the cell is resting. The Goldman equation of weighted averages always applies, but the following approach may be more easily visualized. At any given moment, there are two factors for an ion that determine how much influence that ion will have over the membrane potential of a cell:</p>
<ol style="list-style-type: decimal">
<li>That ion’s driving force</li>
<li>That ion’s permeability</li>
</ol>
<p>If the driving force is high, then the ion is being “pushed” across the membrane. If the permeability is high, it will be easier for the ion to diffuse across the membrane.</p>
<ul>
<li>Driving force is the net electrical force available to move that ion across the membrane. It is calculated as the difference between the voltage that the ion “wants” to be at (its equilibrium potential) and the actual membrane potential (E<sub>m</sub>). So, in formal terms, the driving force for an ion = E<sub>m</sub> - E<sub>ion</sub></li>
<li>For example, at our earlier calculated resting potential of −73 mV, the driving force on potassium is 7 mV : (−73 mV) − (−80 mV) = 7 mV. The driving force on sodium would be (−73 mV) − (60 mV) = −133 mV.</li>
<li>Permeability is a measure of how easily an ion can cross the membrane. It is normally measured as the (electrical) conductance and the unit, siemens (S), corresponds to 1 C·s<sup>−1</sup>·V<sup>−1</sup>, that is one coulomb per second per volt of potential.</li>
</ul>
<p>So, in a resting membrane, while the driving force for potassium is low, its permeability is very high. Sodium has a huge driving force but almost no resting permeability. In this case, potassium carries about 20 times more current than sodium, and thus has 20 times more influence over E<sub>m</sub> than does sodium.</p>
<p>However, consider another case—the peak of the action potential. Here, permeability to Na is high and K permeability is relatively low. Thus, the membrane moves to near E<sub>Na</sub> and far from E<sub>K</sub>.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="neurons-and-glial-cells.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="neurotransmission.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": false,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nikolaussucher/neuroanatomy-text/edit/master/potential.rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
